Common/Core/TrackSelectorPID.h

```diff
@@ -46,7 +46,36 @@ class TrackSelectorPID
     PIDAccepted
   };
 
-  void setPDG(int pdg) { mPdg = std::abs(pdg); }
+  void setPDG(int pdg)
```

Comment:
To make sure that both `mPdg` and `mSpecies` are set in the standard constructor, `setPDG(pdg)` should replace `: mPdg(std::abs(pdg))` there.